test/spec/patient-search/controllers/patient-search.controller.test.js

```diff
@@ -0,0 +1,84 @@
+/*
+ jshint -W098, -W117, -W030
+ */
+(function() {
+  'use strict';
+  describe('Controller: PatientSearchCtrl ', function(){
+    var controller;
+    var scope;
+    var filterFilter;
+    var state;
+    var openmrsRestService;
+    var log;
+
+
+    beforeEach(function(){
+      module('ngAmrsApp');
+      module('app.openmrsRestServices');
+      module('app.patientsearch');
+      module('mock.data');
+
+    });
+
+    beforeEach(inject(function($controller, $injector, $rootScope, OpenmrsRestService){
+
+      //Injecting required dependencies
+      scope =   $rootScope.$new();
+      state = $injector.get('$state');
+      openmrsRestService = $injector.get('OpenmrsRestService');
+      log=$injector.get('$log');
+      filterFilter=$injector.get('filterFilter');
+
+      controller =$controller('PatientSearchCtrl', {
+        $rootScope:scope,
+        OpenmrsRestService:openmrsRestService,
+        $scope:scope,
+        $log:log,
+        filterFilter:filterFilter,
+        $state:state
+
+      });
+
+    }));
+    describe('Patient-Search Controller Unit Tests', function(){
```

Comment:
You don't need another describe here. The it() methods can be called within the first one.